patterns/banner-poster.php

```diff
@@ -21,7 +21,12 @@
 			<!-- wp:column {"width":"80%"} -->
 			<div class="wp-block-column" style="flex-basis:80%">
 				<!-- wp:heading {"textAlign":"left","align":"wide","style":{"typography":{"fontSize":"12vw","lineHeight":"0.9","fontStyle":"normal","fontWeight":"300"}}} -->
-				<h2 class="wp-block-heading alignwide has-text-align-left" style="font-size:12vw;font-style:normal;font-weight:300;line-height:0.9"><?php echo esc_html_x( '“Stories, historias, iсторії, iστορίες”', 'Sample heading in four languages.', 'twentytwentyfive' ); ?></h2>
+				<h2 class="wp-block-heading alignwide has-text-align-left" style="font-size:12vw;font-style:normal;font-weight:300;line-height:0.9">
+					<?php
+						/* translators: This string contains the word "Stories" in four different languages with the first item in the locale's language. */
+						echo esc_html_x( '“Stories, historias, iсторії, iστορίες”', 'Placeholder heading in four languages.', 'twentytwentyfive' );
```

Comment:
would it be possible to replace esc_html_x with a more general KSES function? Because in the current proposal, we can't add any html tag in this string, so we can't comply with accessibility rules that stipulates that languages changes must be indicated, for example via `<span lang="en">Stories</span>`

Also it would be nice to indicate the lang of each of the four words in the translator's comment

Reply:
Yes, `lang` attributes would be good. That's also gonna make the string a bit more difficult to grasp for translators. :S

Indicating the language would also be helpful.

Reply:
Thanks for the input!

> Also it would be nice to indicate the lang of each of the four words in the translator's comment

If we go the KSES route and add a `lang` attribute for each, then would it still make sense to have these strings translatable? What would be the reasoning? As a translator, I'd be really confused.

Reply:
All text needs to be translatable.